assignment3.txt

```diff
@@ -0,0 +1,16 @@
+Olivier Gabison
+
+1. for(;i < 1000;i*=2)
+{
+    printf(“%d\n”);
+}
+
+The stuff that are incorrect is that for the printf, if you use %d, you have to include a variable after
```

Comment:
Correct! The assumption that would be needed is for i to be initialized and greater than 0 so that the program would run.